Gemfile.lock

```diff
@@ -370,8 +370,8 @@ GEM
     http-cookie (1.0.3)
       domain_name (~> 0.5)
     http-form_data (2.3.0)
-    http-parser (1.2.1)
-      ffi-compiler (>= 1.0, < 2.0)
+    http-parser (1.2.2)
```

Comment:
dependency of `http` which is used by the Honeycomb client, some Ruby 2.7 related fixes https://github.com/cotag/http-parser/commits/master